gradle/scripts/scoverage.gradle

```diff
@@ -4,7 +4,7 @@
 
 scoverage {
 	scoverageVersion = "2.3.0"
-	scoverageScalaVersion = scalaBinaryVersion
+	scoverageScalaVersion //= scalaBinaryVersion
```

Comment:
What about this?

Reply:
I think the line can be removed, as the scala version is now automatically detected: https://github.com/scoverage/gradle-scoverage?tab=readme-ov-file#configuration